### PR TITLE
Stop setting AA_EnableHighDpiScaling since Qt Says it is not needed

### PR DESCRIPTION
**Pull request by hmaarrfk**
See https://doc.qt.io/qt-6/highdpi.html

### CODE REVIEW

File: wgpu/gui/qt.py [hunk 1]
```diff
@@ -124,7 +124,14 @@ def enable_hidpi():
         except Exception:
             pass  # fail on non-windows
     try:
-        QtWidgets.QApplication.setAttribute(QtCore.Qt.AA_EnableHighDpiScaling, True)
+        # This flag is not needed on Qt6.6
```
Comment (hmaarrfk) [inline wgpu/gui/qt.py:127]:
```suggestion
        # This flag is not needed on Qt6
```

**Comment by hmaarrfk** [comment]
The warning only shows up in test suites, but it finally got to me mentally....

File: wgpu/gui/qt.py [hunk 2]
```diff
@@ -124,7 +124,14 @@ def enable_hidpi():
...
             pass  # fail on non-windows
     try:
-        QtWidgets.QApplication.setAttribute(QtCore.Qt.AA_EnableHighDpiScaling, True)
+        # This flag is not needed on Qt6
+        # But it has been definitely giving me a warning since PySide 6.6
+        # I'm a little cautious of removing its application for much older
+        # PySide so I'm going to conditionally disable it for "newer" PySide
+        # hmaarrfk -- 2024/12
+        # https://doc.qt.io/qt-6/highdpi.html
+        if qt_version < (6, 6):
```
Comment (hmaarrfk) [inline wgpu/gui/qt.py:133]:
```suggestion
        if qt_version_info < (6, 6):
```

File: wgpu/gui/qt.py [hunk 3]
```diff
@@ -124,7 +124,14 @@ def enable_hidpi():
...
+        # This flag is not needed on Qt6
+        # But it has been definitely giving me a warning since PySide 6.6
+        # I'm a little cautious of removing its application for much older
+        # PySide so I'm going to conditionally disable it for "newer" PySide
+        # hmaarrfk -- 2024/12
+        # https://doc.qt.io/qt-6/highdpi.html
+        if qt_version_info < (6, 6):
+            QtWidgets.QApplication.setAttribute(QtCore.Qt.AA_EnableHighDpiScaling, True)
     except Exception:
         pass  # fail on older Qt's
```
Comment (hmaarrfk) [inline wgpu/gui/qt.py:136]:
```suggestion
        pass  # fail on much older Qt's
```

**Review by Korijn — APPROVED** [review]

**Pull request merged**